components/dashboard/src/components/DropDown2.tsx

```diff
@@ -201,7 +203,13 @@ export const DropDown2: FunctionComponent<DropDown2Props> = ({
                         </div>
                     )}
                     <ul className="max-h-60 overflow-auto">
-                        {filteredOptions.length > 0 ? (
+                        {loading && (
+                            <div className="flex-col space-y-2 animate-pulse">
+                                <div className="bg-gray-300 dark:bg-gray-500 h-4 rounded" />
+                                <div className="bg-gray-300 dark:bg-gray-500 h-4 rounded" />
```

Comment:
What do you suggest we do instead? Previously it would just show "No results" while it was in a loading state and open/focused, so wanted to fix that and this was my attempt. Happy for other ideas though.

Reply:
@selfcontained I'd suggest keeping the skeleton loader only inside the dropdown, but keep the dropdown title as is as there's no need to hide that. What do you think?

Reply:
A challenge is that we can't render the title until we've loaded some data so we can account for if it's a project or not, and render the right name. We should be able to have that data cached so you don't always see a loading state here.

Reply:
I'd think the dropdown _**title**_ would be always _**Select a repository**_ + **_Context URL**_ when the dropdown is expanded, right? Do you think it's better to show a skeleton loader there, too?

Reply:
Let's sync on this sometime - hard to convey the nuances like this, but I'd love to simplify it if we can and remove loaders where possible.

Reply:
DEAL